src/content/docs/apm/agents/ruby-agent/getting-started/migration-9x-guide.mdx

```diff
@@ -0,0 +1,102 @@
+  | `disable_memcached`                       | `instrumentation.memcached`               | `instrumentation.memcached: disabled`                                               |
+  | `disable_mongo`                           | `instrumentation.mongo`                   | `instrumentation.mongo: disabled`                                                   |
+  | `disable_net_http`                        | `instrumentation.net_http`                | `instrumentation.net_http: disabled`                                                |
+  | `prepend_net_instrumentation`             | `instrumentation.net_http`                | `instrumentation.net_http: prepend`                                                 |
+  | `disable_puma_rack`                       | `instrumentation.puma_rack`               | `instrumentation.puma_rack: disabled`                                               |
+  | `disable_puma_rack_urlmap`                | `instrumentation.puma_rack_urlmap`        | `instrumentation.puma_rack_urlmap: disabled`                                        |
+  | `disable_rack`                            | `instrumentation.rack`                    | `instrumentation.rack: disabled`                                                    |
+  | `disable_rack_urlmap`                     | `instrumentation.rack_urlmap`             | `instrumentation.rack_urlmap: disabled`                                             |
+  | `disable_redis`                           | `instrumentation.redis`                   | `instrumentation.redis: disabled`                                                   |
+  | `disable_redis_instrumentation`           | `instrumentation.redis`                   | `instrumentation.redis: disabled`                                                   |
+  | `disable_resque`                          | `instrumentation.resque`                  | `instrumentation.resque: disabled`                                                  |
+  | `disable_sinatra`                         | `instrumentation.sinatra`                 | `instrumentation.sinatra: disabled`                                                 |
+  | `disable_rake`                            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_rake_instrumentation`            | `instrumentation.rake`                    | `instrumentation.rake: disabled`                                                    |
+  | `disable_typhoeus`                        | `instrumentation.typhoeus`                | `instrumentation.typhoeus: disabled`                                                |
+
+
+## Removed Support for Ruby 2.2 and 2.3 [#ruby_2223]
+
+  Ruby 2.2 and 2.3 are no longer supported by the Ruby agent. In order to continue using the latest Ruby Agent version, please update to Ruby 2.4.0 or above.
+
```

Comment:
I think it would be helpful to accompany our announcement of dropping support for Ruby 2.2 and 2.3 with a reminder that we are still supporting Ruby versions that are much older than what the Ruby team officially supports.

Perhaps something like:

> Note that New Relic continues to provide support for certain older Ruby versions even after they have fallen out of the officially supported maintenance window provided by the Ruby language development team. For example, as of March 2023 the oldest Ruby version supported by the Ruby team will be version 3.0, whereas with version 9.0 of the New Relic agent, Ruby versions 2.4 and above are still supported. For more information about official Ruby team support of Ruby versions, see [Ruby Maintenance Branches](https://www.ruby-lang.org/en/downloads/branches/).

Reply:
I like this! [Added](https://github.com/newrelic/docs-website/pull/11408/commits/e806ec4b072854c839ba264913a59cd002b41b0a).